src/Solver/time_stepping/TimeManager.cpp

```diff
@@ -134,7 +136,8 @@ void seissol::time_stepping::TimeManager::addClusters(TimeStepping& i_timeSteppi
           memoryManager.getFaultOutputManager(),
           seissolInstance,
           &m_loopStatistics,
-          &actorStateStatisticsManager.addCluster(profilingId))
+          &actorStateStatisticsManager.addCluster(profilingId),
+          &damagedelasticparameters)
```

Comment:
No need to pass damageparameters, can be accessed through `seissolInstance.getSeisSolParameters()...`